app/src/main/java/org/mozilla/fenix/search/awesomebar/AwesomeBarUIView.kt

```diff
@@ -59,7 +59,6 @@ class AwesomeBarUIView(
                     if (useNewTab) {
                         components.useCases.searchUseCases.newTabSearch
                     } else components.useCases.searchUseCases.defaultSearch,
-                    components.core.client,
```

Comment:
Drive-by comment: We definitely don't want to remove this and will break the build on the latest snapshot.

Reply:
Thanks for the heads up. Must have been a merging issue :)